app/javascript/utilities/showModal.jsx

```diff
@@ -37,6 +37,7 @@ export const showWindowModal = async ({
   modalContent,
   contentSelector,
   onOpen,
+  document = window.document,
```

Comment:
Can we add this new paramter to the JSDoc notes above the function? Just to let folks know that the reason for using this prop would likely be to account for iframes. I think the 'type' can just be `Object`